test/abi/macOS/arm64/stdlib-asserts.swift

```diff
@@ -59,4 +59,13 @@ Added: _$ss20__StaticArrayStorageCfd
 Added: _OBJC_CLASS_$__TtCs20__StaticArrayStorage
 Added: _OBJC_METACLASS_$__TtCs20__StaticArrayStorage
 
+// Swift.String.init<A, B where A: Swift._UnicodeEncoding, B: Swift.Sequence, A.CodeUnit == B.Element>(validating: B, as: A.Type) -> Swift.String?
```

Comment:
You can remove these additions in the assertion tests, it picks up all the exceptions in the non-assert one.

Reply:
Done, thanks for the clarification.